samples/event-sourced-customer-registry/src/main/java/customer/application/CustomerEntity.java

```diff
@@ -56,6 +56,7 @@ public Effect<Done> changeAddress(Address newAddress) {
 
   @Override
   public Customer applyEvent(CustomerEvent event) {
+    System.out.println("test event: " + event);
```

Comment:
runaway println

Reply:
yeah, testing to many things at the same time :)